efs-lib/src/cmr/efs/connection.clj

```diff
@@ -2,16 +2,39 @@
   "Contains functions for interacting with the EFS storage instance."
   (:require
    [clojure.java.io :as io]
+   [clojure.string :as str]
    [clj-time.coerce :as cr]
    [cmr.common.log :refer [debug error info trace warn]]
+   [cmr.common.util :as util]
    [cmr.efs.config :as efs-config]
    [cmr.common.log :refer (debug info warn error)]
    [cmr.common.services.errors :as errors]
    [cmr.common.services.health-helper :as hh])
   (:import
    (java.nio.file Files
                   Paths
-                  OpenOption)))
+                  OpenOption)
+   (java.io File)))
+
+;;--------------------- UTILITY FUNCTIOINS ---------------------
+(defn get-revision-file-names
+  "Returns a listing of revisions of the concept stored on EFS"
+  [provider concept-type concept-id]
+  (let [concept-dir-path (format "%s/%s/%s/%s" (efs-config/efs-directory) provider concept-type concept-id)]
+    (map (fn [file]
+           (.getName file)) (.listFiles (File. concept-dir-path)))))
+
+(defn get-latest-revision
+  [provider concept-type concept-id]
+  (first (sort > (map (fn [file-name]
+                        (Integer/parseInt (subs (second (str/split file-name #"\.")) 1))) (get-revision-file-names provider concept-type concept-id)))))
```

Comment:
Can you please put the (get-revision-file-name ... on the next line, that lines up with (fn ...) since this is another parameter to the map function.